File: Versão em JavaScript/Algoritmos - Lista/Exercício - 7.js
// Exercicio - 7
// Elabore um algoritmo que dada a idade de um nadador classifica - o em uma das seguintes
// categorias:
// infantil A = 5 - 7 anos
// infantil B = 8 - 10 anos
// juvenil A = 11 - 13 anos
// juvenil B = 14 - 17 anos
// adulto = maiores de 18 anos

const prompt = require('prompt-sync')({ sigint: true })

const x = prompt('Digite a idade do nadador: ')
console.log('================================')

idade = (Number(x))

if (idade <= 7) {
    console.log('O nadador pertence a categoria Infantil A')
}
else if (idade <= 10) {
    console.log('O nadador pertence a categoria Infantil B')
}
else if (idade <= 13) {
    console.log('O nadador pertence a categoria Juvenil A')
}
else if (idade <= 17) {
    console.log('O nadador pertence a categoria Juvenil B')
}
else {
    console.log('O nadador é adulto')
}

File: Versão em Python/Algoritmos - Lista/Exercício - 7.py
# Exercicio - 7
# Elabore um algoritmo que dada a idade de um nadador classifica - o em uma das seguintes
# categorias:
# infantil A = 5 - 7 anos
# infantil B = 8 - 10 anos
# juvenil A = 11 - 13 anos
# juvenil B = 14 - 17 anos
# adulto = maiores de 18 anos

x = input('Digite a idade do nadador: ')
idade = int(x)

if idade <= 7:
    print('O nadador pertence a categoria Infantil A ')
elif idade <= 10:
    print('O nadador pertence a categoria Infantil B ')
elif idade <= 13:
    print('O nadador pertence a categoria Juvenil A ')
elif idade <= 17:
    print('O nadador pertence a categoria Juvenil B ')
else:
    print('O nadador é adulto')
